releases/mainnet/1.9.3/index.html

```diff
@@ -0,0 +1,27 @@
+<html>
+	<head>
+		<title>Mainnet Release 1.9.3</title>
+	</head>
+	<body>
+		<div>
+			<h2>
+				<strong>Mainnet Release 1.9.3</strong>
+			</h2>
+
+			<ul>
+				<li><a href="/releases/mainnet/1.9.3/contracts.json">Contract addresses</a></li>
+				<li><a href="https://ipfs.io/ipfs/QmP1HLPtmE2C8WNpwEXAyFi1LVc3PnT4QLmyCNJxUSCBKj">Contract addresses (IPFS hosted)</a></li>
+				<li><a href="/releases/mainnet/1.9.3/abi/index.html">ABIs</a></li>
+				<!--
```

Comment:
@gbalabasquer we will not have a tag for this release either so having this commented out is correct, right?